phpmyfaq/admin/assets/scss/_variables.scss

```
@@ -1,31 +1,31 @@
 //
```

Stylelint (17.11.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)